pkgs/top-level/all-packages.nix

```diff
@@ -7750,7 +7744,7 @@ with pkgs;
     inherit (darwin.apple_sdk_11_0.frameworks) CoreFoundation CoreServices Foundation IOKit;
     buildJdk = jdk21_headless;
     runJdk = jdk21_headless;
-    stdenv = if stdenv.hostPlatform.isDarwin then darwin.apple_sdk_11_0.stdenv
+    stdenv = if stdenv.hostPlatform.isDarwin then stdenv
       else if stdenv.cc.isClang then llvmPackages.stdenv
       else stdenv;
```

Comment:
`if stdenv.cc.isClang && !stdenv.hostPlatform.isDarwin`? (But also: What on earth is this doing? Probably it can just be dropped?)

Reply:
Tracked it down to f863703dd2966. This is doing nothing useful and can just be dropped.

Reply:
I'll do it in #370672. Currently, all Bazel versions fail to build on darwin, I guess some LLVM overrides will be needed: https://hydra.nixos.org/eval/1810825?filter=bazel&compare=1810818&full=#tabs-still-fail